mangaki/mangaki/utils/mal.py

```diff
@@ -46,8 +47,8 @@ class MALWorks(Enum):
 
 SUPPORTED_MANGAKI_WORKS = [MALWorks.animes, MALWorks.mangas]
 
-
-MALUserWork = namedtuple('MALUserWork', 'title poster mal_id score')
+MALUserWork = namedtuple('MALUserWork', 'title synonyms poster mal_id score')
+MALUserWork.__hash__ = lambda work: hash(work.title)
```

Comment:
I am just afraid that `mal_id` is not unique :/.
I need to double-check.

As for the alternative to the `namedtuple`, I could understand this is ugly, but it's a lot less verbose than creating a whole full fledged class, don't you think?